src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator: Excavator started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator: Excavator stopped working");

```

Comment:
The stop message should also be more informative. Use the class name to indicate which machine stopped working. For example: 'Excavator has stopped working.'